docs/tutorials/data_requirements.rst

```diff
@@ -150,7 +123,7 @@ Target
 
 The actual outcome of the event the machine learning model is trying to predict.
```

Comment:
`/_static/tutorials/data-requirements-index-based-x-axis.svg` and `/_static/tutorials/data-requirements-time-based-x-axis.svg` are need to be refreshed.